src/client/src/containers/About/index.tsx

```diff
@@ -46,17 +46,6 @@ const About = ({ versions, intl }: Props) => {
           Web Template Studio
         </a>
       </p>
-      <p>
-        <a
-          className={styles.link}
-          href="https://github.com/Microsoft/WebTemplateStudio/issues"
-        >
-          <FormattedMessage
```

Comment:
Feels like we can remove FormattedMessage from imports as well :)

Reply:
Agreed. There were a couple of other unused imports as well. Refactored it. :)